apps/aecore/lib/aecore/tx/signed_tx.ex

```diff
@@ -50,7 +50,7 @@ defmodule Aecore.Tx.SignedTx do
 
   @spec validate(SignedTx.t(), non_neg_integer()) :: :ok | {:error, String.t()}
   def validate(%SignedTx{data: data} = tx, block_height) do
-    if signatures_valid?(tx) do
+    if DataTx.chainstate_senders?(data) || signatures_valid?(tx, DataTx.senders(data)) do
```

Comment:
Will this break anything? Skipping the signature check when the pubkeys are not available does not seem right. Could the tx verification logic pass the chainstate here so the verification could proceed?

Reply:
Chainstate is not available by definition in `validate` functions. Preprocess check is for checks requiring chainstate.